create-topics.sh

```diff
@@ -35,7 +35,14 @@ IFS=','; for topicToCreate in $KAFKA_CREATE_TOPICS; do
     if [ -n "${topicConfig[3]}" ]; then
         config="--config=cleanup.policy=${topicConfig[3]}"
     fi
-    JMX_PORT='' "$KAFKA_HOME/bin/kafka-topics.sh" --create --zookeeper "$KAFKA_ZOOKEEPER_CONNECT" --replication-factor "${topicConfig[2]}" --partitions "${topicConfig[1]}" --topic "${topicConfig[0]}" "$config" --if-not-exists &
+    for i in {4..100}
```

Comment:
This seems a little inconsistent now to set topic configuration in two separate ways. Is this really needed?

Reply:
I agree. I was thinking about backward compatibility when doing it this way. Removing the first 3 hard coded config keys would be more consistent though.

Reply:
I think the first 3 make sense, as they are the minimum topic configuration (name / partitions / replication factor). However, the others are all advanced config, so makes sense to call out specifically in key/value format. (i.e. anything that gets passed to `--config`).

I recently added an 'Announcement' section to the README which has all breaking changes in, and there is also a section explaining how `KAFKA_CREATE_TOPICS` works (https://github.com/wurstmeister/kafka-docker#automatically-create-topics) - this should also be updated to reflect any changes. Thanks. 

If you don't have the inclination, I can always tidy up the README/CHANGELOG etc later.